tests/libkubevirt/config/kvconfig.go

```diff
@@ -163,21 +173,32 @@ func WaitForConfigToBePropagatedToComponent(podLabel string, resourceVersion str
 	}, duration, 1*time.Second).ShouldNot(HaveOccurred())
 }
 
-func callUrlOnPod(pod *k8sv1.Pod, port string, url string) ([]byte, error) {
-	randPort := strconv.Itoa(4321 + rand.Intn(6000))
+func callURLOnPod(pod *k8sv1.Pod, port, url string) ([]byte, error) {
+	const minPort = 4321
+	const maxPortIncrease = 6000
+	//nolint:gosec
```

Comment:
Just an alternative:
```go
	randPort, err := rand.Int(rand.Reader, new(big.Int).SetInt64(maxPortIncrease))
	if err != nil {
		return nil, err
	}
	randPort.Add(new(big.Int).SetInt64(minPort), randPort)
```
And use `randPort.String()` in the Sprintf

Reply:
math/rand doesn't have Reader